index.html

```diff
@@ -3,416 +3,652 @@
     <script type="text/javascript" src="common/common.js" class="remove" defer></script>
     <script type="text/javascript" src="common/jsonld.js" class="remove"></script>
     <script class='remove'>
       var respecConfig = {
         localBiblio: jsonld.localBiblio,
-        specStatus: "ED",
+        specStatus: "WG-NOTE",
         copyrightStart:    "2019",
         shortName:  "json-ld-bp",
         edDraftURI: "https://w3c.github.io/json-ld-bp/",
-        github:            "https://github.com/w3c/json-ld-bp/",
+        github:     "https://github.com/w3c/json-ld-bp/",
         editors: [{
           name: "Gregg Kellogg",
           url: "http://greggkellogg.net/",
           w3cid: "44770"
         }],
-        //previousMaturity: "FPWD",
+        //previousMaturity: "NOTE",
         //previousPublishDate: "1977-03-15",
-        wg:           "JSON-LD Working Group",
-        wgURI:        "https://www.w3.org/2018/json-ld-wg/",
+        github:     "https://github.com/w3c/json-ld-bp/",
+        doJsonLd:   true,
         wgPublicList: "public-json-ld-wg",
-        maxTocLevel:        4,
-        noRecTrack:         "true"
+        maxTocLevel:  4,
+        noRecTrack:   true
       };
     </script>
+    <style>
+      .hl-bold { font-weight: bold; color: #0a3; }
+      .comment { color: #999; }
+    </style>
   </head>
-  <body>
-    <section id='abstract'>
-      <p>
-        Developers share a common problem: they want a simple, but extensible way to create an API for a web service that gets the job done, doesn't design them into a corner, and allows developers to easily interact with their service without reinventing the wheel. JSON-LD [[JSON-LD]] has become an important solution, as it bridges the gap between formally data and more colloquial JSON interfaces used in APIs from numerous providers. This guide attempts to define certain best practices for publishing data using JSON-LD, and interacting with such services.
-      </p>
-    </section>
+<body>
+<section id='abstract'>
+  <p>
+    Developers share a common problem: they want a simple,
+    but extensible way to create an API for a web service that gets the job done,
+    doesn't design them into a corner,
+    and allows developers to easily interact with their service without reinventing the wheel.
+    JSON-LD [[JSON-LD]] has become an important solution,
+    as it bridges the gap between formally data
+    and more colloquial JSON interfaces used in APIs from numerous providers.
+    This guide attempts to define certain best practices for publishing data using JSON-LD,
+    and interacting with such services.
+  </p>
+</section>
 
-    <section id='sotd'>
-      <p>
-        This is an unofficial proposal.
-      </p>
-    </section>
+<section id='sotd'>
+<p>This unofficial document has been developed by the
+  <a href="https://www.w3.org/2018/json-ld-wg/">JSON-LD Working Group</a>.</p>
```

Comment:
Based on other sections, looks like this needs an indent:
```suggestion
  <p>This unofficial document has been developed by the
    <a href="https://www.w3.org/2018/json-ld-wg/">JSON-LD Working Group</a>.</p>
```